packages/app/src/specs/CreateSpecModal.vue

```diff
@@ -95,7 +96,11 @@ const helpLink = computed(() => {
   return ''
 })
 
-const specFileName = computed(() => getPathForPlatform('cypress/e2e/filename.cy.js'))
+const specFileName = computed(() => {
+  const extension = props.gql.currentProject?.hasTypescript ? 'ts' : 'js'
```

Comment:
Fine for now, but we have a ternary like `hasTypescript ?` in many places. I wonder if we eventually want a getter, like

```ts
get fileExtension () {
  return this.ctx.lifecycleManager.hasTypescript ? 'ts' : 'js'
}
```

Might be good to keep in mind.

![image](https://user-images.githubusercontent.com/19196536/158098964-6b9f62de-bb7e-43da-a2ed-ff5dd4a7bf96.png)